File: Whist/src/dev/tgpgamez/whist/Stats.java
package dev.tgpgamez.whist;

public class Stats {

    private String username;
    public String getUsername() {
        return username;
    }

    private int wins;
    public int getWins() {
        return wins;
    }

    private int lost;
    public int getLost() {
        return lost;
    }

    private int gamesPlayed;

    public int getGamesPlayed() {
        return gamesPlayed;
    }

    public Stats() {
        this.username = "None";
        this.wins = 0;
        this.lost = 0;
        this.gamesPlayed = 0;
    }

    public Stats(String username, int wins, int lost, int gamesPlayed) {
        this.username = username;
        this.wins = wins;
        this.lost = lost;
        this.gamesPlayed = gamesPlayed;
    }

    @Override
    public String toString() {
        return "Stats{" +
                "username='" + username + '\'' +
                ", wins=" + wins +
                ", lost=" + lost +
                ", gamesPlayed=" + gamesPlayed +
                '}';
    }
}

File: Whist-Version-js/Database/Models/Stats.js
"use strict";
exports.__esModule = true;
exports.Stats = void 0;
var Stats = /** @class */ (function () {
    function Stats(username, wins, lost, gamesPlayed) {
        this.username = username;
        this.wins = wins;
        this.lost = lost;
        this.gamesPlayed = gamesPlayed;
    }
    Stats.prototype.GetUsername = function () {
        return this.username;
    };
    Stats.prototype.GetWins = function () {
        return this.wins;
    };
    Stats.prototype.GetLost = function () {
        return this.lost;
    };
    Stats.prototype.GetGamesPlayed = function () {
        return this.gamesPlayed;
    };
    return Stats;
}());
exports.Stats = Stats;
